employees/tests/test_unit_templatetags.py

```diff
@@ -0,0 +1,16 @@
+from django.test import TestCase
+
+from employees.templatetags.data_structure_element_selectors import get_key_value
+
+
+class GetKeyValueTests(TestCase):
+    def setUp(self):
+        self.dictionary = {"one": 1, "two": 2, "three": 3}
+
+    def test_get_key_value_function_should_return_an_element_of_given_dict_at_given_key(self):
+        result = get_key_value(self.dictionary, "two")
+        self.assertEqual(result, 2)
+
+    def test_get_key_value_function_should_handle_key_error(self):
```

Comment:
If you apply https://github.com/Code-Poets/sheetstorm/pull/137/files#r284477996 pelase remember to change test name.